packages/commerce-sdk-react/README.md

```diff
@@ -64,7 +101,259 @@ const App = ({children}) => {
 
 export default App
 ```
-### Useful Links:
+
+## Shopper Authentication and Token Management
+
+_💡 This section assumes you have read and completed the [Authorization for Shopper APIs](https://developer.salesforce.com/docs/commerce/commerce-api/guide/authorization-for-shopper-apis.html) guide._
+
+To help reduce boilerplate code for managing shopper authentication, by default, this library automatically initializes shopper session and manages the tokens for developers. Currently, the library supports the [Public Client login flow](https://developer.salesforce.com/docs/commerce/commerce-api/guide/slas-public-client.html).
+
+### Shopper Session Initialization
+
+On `CommerceApiProvider` mount, the provider initializes shopper session by initiating the SLAS Public Client login flow. The tokens are stored/managed by the library to support Phased Launch use cases and multi-site setup.
+
+When the access token is expired, the library will automatically try to refresh the access token by calling the `/oauth2/token` endpoint with `grant_type=refresh_token`.
+
+### Authenticate request queue
+
+While the library is getting/refreshing the access token, the network requests will queue up until there is a valid access token.
+
+### Auth helpers
+
+To leverage the managed shopper authentication feature, use the auth helper hooks for shopper login.
+
+Example:
+
+```jsx
+import {AuthHelpers, useAuthHelper} from '@salesforce/commerce-sdk-react'
+
+const Example = () => {
+    const register = useAuthHelper(AuthHelpers.Register)
+    const login = useAuthHelper(AuthHelpers.LoginRegisteredUserB2C)
+    const logout = useAuthHelper(AuthHelpers.LogOut)
+
+    return <button onClick={() => {
+        login.mutate({username, password})
+    }}>
+}
+```
+
+### Opt-out from managed shopper authentication
+
+To opt-out of the auto-login, request queue, and token management features, pass `fetchedToken` (a SLAS JWT) to the provider.
+
+```jsx
+const MyComponent = ({children}) => {
+    return (
+        <CommerceApiProvider
+            fetchedToken="xxxxxxxxxxxx"
+        >
+            {children}
+        </CommerceApiProvider>
+    )
+} 
+```
+
+## Queries
```

Comment:
I feel like we have these part headings for big topics like "Queries" and "Mutations" .. but we don't have a segue into them. Do we need to add something here, like: 

"The query hooks library can be broken down into queries, things that get data, and mutations, things that change data. Below are the basics of using mutations and queries: "

Reply:
added segue for react-query intro